include/nccl_ofi_tuner.h

```diff
@@ -0,0 +1,91 @@
+#ifndef NCCL_OFI_TUNER_H_
+#define NCCL_OFI_TUNER_H_
+
+#include <linux/limits.h>
+#include <float.h>
+#include "nccl-headers/tuner.h"
+
+/*
+ * TODO: This should not be statically defined. The plugin interface lets us
+ * tune the number of channels as well, but that can come later (once a
+ * proto+algo combination is chosen, we can compute the cost with different
+ * channel count and optimize for it.
+ */
+#define NCCL_OFI_TUNER_NUM_CHANNELS	(8)
+
+/* Latency in µsecs and bandwidths in Bytes/µsec */
+#define NCCL_OFI_TUNER_NET_LATENCY	(20)
+#define NCCL_OFI_TUNER_INTRANODE_BW	(12.5 * 1024 * 1024 * 1024 * 1e-6) /* per rail */
+#define NCCL_OFI_TUNER_INTERNODE_BW	(31.5 * 1024 * 1024 * 1024 * 1e-6) /* PCI gen4 x16 baseline */
```

Comment:
gen4x16 or gen5x8 baseline

Reply:
Code already exists in the plugin to compute this on instance, I'll move to using that instead.